org.eclipse.jdt.junit5.runtime/build.properties

```diff
@@ -18,3 +18,4 @@ bin.includes = about.html,\
                plugin.properties,\
                .,\
                META-INF/
+jre.compilation.profile = JavaSE-1.8
```

Comment:
Why change BREE but add a 1.8 compilation profile? If that is intended, please document via commit message.

Reply:
> Why change BREE but add a 1.8 compilation profile? If that is intended, please document via commit message.

See https://bugs.eclipse.org/bugs/show_bug.cgi?id=574363

Reply:
Please document via the commit message.